tbvaccine/tbv.py

```diff
@@ -249,7 +249,8 @@ def format_exc(self):
 
 
 def add_hook(*args, **kwargs):
-    if not getattr(sys.stderr, "isatty", lambda: False)():
-        return
+    if os.getenv('TBVACCINE_FORCE') != '1':
```

Comment:
Could you turn these two `if`s into an `and` for better readability?

Reply:
At first I wrote so but it exceeds 79 chars and needs wrapping:

```py
    if os.getenv('TBVACCINE_FORCE') != '1' and \
            getattr(sys.stderr, "isatty", lambda: False)():
        return
    
    # apply De Morgan's laws:
    
    if not (os.getenv('TBVACCINE_FORCE') != '1' or
            getattr(sys.stderr, "isatty", lambda: False)()):
        return
```

I considered it doesn't improves readability but loses debuggability.

Which do you prefer?

Reply:
And this is why I opened #35:

```py
def add_hook(*args, **kwargs):
    if not (os.getenv('TBVACCINE_FORCE') != '1' or sys.stderr.isatty()):
        return
```

Reply:
Hmm yeah, might as well do the latter and implement #35 as well, I guess these old Python versions aren't relevant any more. I don't like DeMorgan's applied here because it makes things harder to understand (especially with both the `!= 1` and the `not` in there).

Case in point, it took you two edits to write and it's still wrong :P

```
if not (os.getenv("TBVACCINE_FORCE") == "1" or sys.stderr.isatty()):
    return
```

is the correct one, but I prefer:

```
if os.getenv("TBVACCINE_FORCE") != "1" and not sys.stderr.isatty():
    return
```

As for the formatting, just run `black` on the whole thing.

Reply:
> Case in point, it took you two edits to write and it's still wrong :P

Oops, thank you. Edited again.

OK, let's go with it.